media/documents/file.txt

```diff
@@ -0,0 +1 @@
+A boring example document
```

Comment:
rogue file.

could you please add `/media` to `.gitignore` ?

Reply:
added.